client/src/App.js

```diff
@@ -10,6 +10,7 @@ import Navbar from "./component/Navbar";
 import Footer from "./component/Footer";
 import Home from "./screen/Home";
 import NewNoteScreen from "./screen/NewNoteScreen";
+import AboutUs from "./component/AboutUs"
```

Comment:
According to project structure all the pages are located in screen. I would prefer to remove AboutUs file to screen folder

Thank u

Reply:
I believe about us is still required in the footer

Reply:
The about us in footer redirect to a blank page for now